examples/dyn_create_destroy_apps/README.md

```diff
@@ -0,0 +1,21 @@
+# Dynamic app creation and destruction example
```

Comment:
One thing, please update the examples table in the `examples/README.md` file.

Reply:
I feel like I should come up with a better name for the example... Do you have a suggestion?

Reply:
Or maybe not